kafka-commons/src/main/java/uk/gov/homeoffice/digital/sas/kafka/constants/Constants.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.homeoffice.digital.sas.kafka.constants;
+
+public class Constants {
+
+  private Constants() {
+
+  }
```

Comment:
Do you need this constructor?

Reply:
I believe that's done to prevent instantiation. But now that you bring it up, could throw an `AssertionsError` to make it's purpose clear.
(As mentioned in Effective Java book: https://www.informit.com/articles/article.aspx?p=1216151&seqNum=4)

Reply:
Sonar raises this as an issue. Having a private constructor overrides the implicit public one.
![image](https://user-images.githubusercontent.com/109532529/221868874-6e665a39-6530-4baf-8221-f8ef95ed1305.png)

Reply:
Thrown assertion as suggested.

Reply:
thanks, yep looks good